mod-kb-ebsco-java/src/main/resources/domain/mod-kb-ebsco-java/features/resources.feature

```diff
@@ -4,67 +4,250 @@ Feature: Resources
     * url baseUrl
     * callonce login testUser
     * configure headers = { 'Content-Type': 'application/vnd.api+json', 'x-okapi-token': '#(okapitoken)', 'Accept': 'application/vnd.api+json' }
+    * def samplesPath = 'classpath:domain/mod-kb-ebsco-java/features/samples/resources/'
 
-  @Undefined
-  Scenario: POST Resources with 200 on success
-    * print 'undefined'
-
-  @Undefined
-  Scenario: POST Resources should return 400 if Package and provider id are not provided
-    * print 'undefined'
-
-  @Undefined
-  Scenario: POST Resources should return 404 if Title not found
-    * print 'undefined'
+    * def credentialId = karate.properties['credentialId']
+    * def packageWithoutTitleId = karate.properties['packageId']
+    * def packageId = karate.properties['titlesPackageId']
+    * def titleId = karate.properties['titleId']
 
-  @Undefined
-  Scenario: POST Resources should return 422 if Package id is invalid
-    * print 'undefined'
-
-  @Undefined
+  @Positive
+  Scenario: POST Resources with 200 on success
+    Given path '/eholdings/resources'
+    And request read(samplesPath + 'resources.json')
+    When method POST
+    Then status 200
```

Comment:
shouldn't it be 201 on post?

Reply:
It should.
But mod-kb-ebsco returns 200 on post requests: [Api Documentation](https://s3.amazonaws.com/foliodocs/api/mod-kb-ebsco-java/resources.html#eholdings_resources_post)